libsolidity/formal/CHC.cpp

```diff
@@ -994,37 +991,36 @@ smtutil::Expression CHC::summary(FunctionDefinition const& _function)
 	return summary(_function, *m_currentContract);
 }
 
-unique_ptr<smt::SymbolicFunctionVariable> CHC::createBlock(ASTNode const* _node, string const& _prefix)
+Predicate const* CHC::createBlock(ASTNode const* _node, string const& _prefix)
 {
 	auto block = createSymbolicBlock(sort(_node),
 		"block_" +
-		uniquePrefix() +
-		"_" +
-		_prefix +
-		predicateName(_node));
+			uniquePrefix() +
+			"_" +
+			_prefix +
+			predicateName(_node),
```

Comment:
Maybe can be done in one line, updating this one and deleting `"block_" +`
```suggestion
		"block_" + uniquePrefix() + "_" + _prefix + predicateName(_node),
```
But, I am fine with multi-line version if you prefer it more for readability.